docs/configuration.md

```diff
@@ -1103,10 +1103,10 @@ Apart from these, the following properties are also available, and may be useful
     The number of cores to use on each executor.
 
     In standalone and Mesos coarse-grained modes, setting this
-    parameter allows an application to run multiple executors on the
-    same worker, provided that there are enough cores on that
-    worker. Otherwise, only one executor per application will run on
-    each worker.
+    parameter allows an application to launch multiple executors on the
+    same worker in one single schedule iteration, provided that there are enough
+    cores on that worker. Otherwise, only one executor per application
+    will be scheduled on each worker during one single schedule iteration.
```

Comment:
I'm not sure how to address it better, but my thinking is that instead of trying to address it clearly with limited words, leave the corner case as not to say might be better. Just my two cents.